noxfile.py

```diff
@@ -108,7 +108,7 @@ def mypy(session: Session) -> None:
     """Type-check using mypy."""
     args = session.posargs or ["src", "tests", "docs/conf.py"]
     session.install(".")
-    session.install("mypy", "pytest")
+    session.install("mypy", "pytest", "pytest-asyncio", "aioresponses")
```

Comment:
I'm not sure pytest-asyncio and aioresponses are nedded here.

Reply:
Removed from mypy - still needed for tests and typeguard